rally-domain/src/main/groovy/yakworks/rally/orgs/repo/AbstractOrgRepo.groovy

```diff
@@ -98,6 +98,7 @@ abstract class AbstractOrgRepo extends LongIdGormRepo<Org> {
             // do contact, support keyContact for legacy and Customers
             def contactData = data.contact ?: data.keyContact
             if (contactData) createOrUpdatePrimaryContact(org, contactData as Map)
+            createOrUpdatePartitionOrg(org, args.bindAction)
```

Comment:
see comment on why we do location and contact here. PartitionOrg does not belong here.

Reply:
@basejump if we do in afterPersist - thn we cant check, if num/name are dirty/updated so we can keep in sync